### DISCUSSION TITLE
Rails new mySQL adapter "trilogy" getting SSL error

### DISCUSSION THREAD

**Discussion by JudeLawrence** [planetscale/discussion#617] [Q&A] [answered]
Hi, I'm not that well-versed when it comes to this type of stuff and I haven't been able to figure out a fix for this.

I'm getting an error "trilogy_ssl_upgrade: SSL Error: certificate verify failed" when I try using trilogy (the new SQL adapter for Rails). I get this on my local machine and on my hosted App (on fly.io).

Has anyone tried using trilogy with Planetscale yet, or can anyone point my the direction of a possible solution.

**Comment by mscoutermarsh** [comment, marked as answer]
In your database.yml. Set:
```
    ssl_mode: <%= Trilogy::SSL_VERIFY_IDENTITY %>
```

Our own API uses Rails + Trilogy.
